sherpa/ui/utils.py

```diff
@@ -329,6 +332,116 @@ def reduce_ufunc(func):
 copy_reg.pickle(numpy.ufunc, reduce_ufunc)
 
 
+###############################################################################
+#
+# I/O routines
+#
+###############################################################################
+
+
+def read_template_model(modelname, templatefile,
+                        sep=' ', comment='#',
+                        method=sherpa.utils.linear_interp,
+                        template_interpolator_name='default'):
+    """Read in a set of templates and create a template model.
+
+    Parameters
+    ----------
+    modelname : str
+       The identifier for this table model.
+    templatefile : str
+       The name of the file to read in. This file lists the template
+       data files.
```

Comment:
Should we say  that templatefile reads the template index file with the parameters and data files.